app/src/main/res/layout-sw600dp/story_fragment.xml

```diff
@@ -36,13 +37,20 @@
         app:navigationContentDescription="@string/navigate_up"
         app:navigationIcon="?attr/homeAsUpIndicator">
 
+        <asia.ivity.android.marqueeview.MarqueeView
+          android:id="@+id/marquee_view"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          marquee:speed ="10"
```

Comment:
Nit: extra space before the ``=``. Ditto in several of the other changed layout files.

Reply:
Noted.